cmd/down.go

```diff
@@ -83,15 +88,17 @@ func Down() *cobra.Command {
 						return err
 					}
 					selector := utils.NewOktetoSelector("Select which development container to deactivate:", "Development container")
-					dev, err = utils.SelectDevFromManifest(manifest, selector)
+					options, err := apps.ListDevModeOn(ctx, manifest.Namespace, c)
+					if err != nil {
+						return err
+					}
+					if len(options) < 1 {
+						oktetoLog.Success("All development containers are deactivated")
+					}
+					dev, err = utils.SelectDevFromManifest(manifest, selector, options)
```

Comment:
Two things:
1.- We should show the name of the manifest instead of the name of the kubernetes resource because we add the suffix `-okteto` and that can cause some friction with the users
2.- I think there is an use case where this may not work. Imagine that I have two projects on the same namespace:
**okteto.yaml**
```yaml
name: a
image: alpine
command: sh
sync:
- .:/usr/src/app
forward:
- 9090:8080
autocreate: true
```
and **oktetov2.yml**
```yaml
dev:
  b:
    image: alpine
    command: sh
    sync:
    - .:/usr/src/app
    forward:
    - 7121:8080
    autocreate: true
  c: 
    image: alpine
    command: sh
    sync:
    - .:/usr/src/app
    forward:
    - 7121:8080
    autocreate: true
```
If I run `okteto up -f okteto.yml` and on another terminal okteto up -f `oktetov2.yml` when I run `okteto down -f oktetov2.yml` I should only receive the services defined on the `oktetov2.yml`, but I'm not seeing a filtering over the services that are on dev mode

Reply:
i've changed the function so it retrieves resources from dev manifest. The name will be always the one of the dev manifest as this resource is the one with the dev mode label. The additional resource created with `-okteto` is the dev mode resource which does not have the label.

also thnks to this example, i've added logic to be aware of autocreate resources, which will have the suffix